### ISSUE TITLE
Setting random_state in iterative_train_test_split

### ISSUE THREAD

**Issue by MiladShahidi** [scikit-multilearn/scikit-multilearn#147]
For the sake of reproducibility (and convenience), it would be useful if one could pass `random_state` to `iterative_train_test_split` as an argument, to set the random_state of the internal `IterativeStratification` object.

**Comment by AlJohri** [comment]
I would love to see this as well. Looks like is already a PR for it: https://github.com/scikit-multilearn/scikit-multilearn/pull/141/files

@niedakh can you merge this?

**Comment by MarshallCN** [comment]
You can use `np.random.seed(0)` to set a random seed before the `iterative_train_test_split` , which also makes the results reproducible.

**Comment by florianbaer** [comment]
Is it possible that the iterative_train_test_split is not reproducible when setting the seed? I encounter different splits when running multiple times 🤔 
**disclaimer:** _I run the script on the main branch_
